src/CommonLib/LDAPUtils.cs

```diff
@@ -713,9 +714,8 @@ public IEnumerable<ISearchResultEntry> QueryLDAP(LDAPQueryOptions options)
                 {
                     if (le.ErrorCode != 82)
                         if (throwException)
-                            throw new LDAPQueryException(string.Format(
-                                "LDAP Exception in Loop: {0}. {1}. {2}. Filter: {3}. Domain: {4}.",
-                                le.ErrorCode, le.ServerErrorMessage, le.Message, ldapFilter, domainName), le);
+                            throw new LDAPQueryException(
+                                $"LDAP Exception in Loop: {le.ErrorCode}. {le.ServerErrorMessage}. {le.Message}. Filter: {ldapFilter}. Domain: {domainName}.", le);
```

Comment:
I knew there had to be a better way to handle formatting these strings, I'll remember this for future interpolation needs.